### ISSUE TITLE
False Positive: lowercase attribute

### ISSUE THREAD

**Issue by sk-** [deezer/html-linter#3]
When an attribute is a number (product of being replaced by the tempalte remover),  a CapitalizationMessage will be reported. This is because python islower is broken (see http://bugs.python.org/issue13822).

```
In [1]: '0'.islower()
Out[1]: False

In [2]: '0' == '0'.lower()
Out[2]: True
```
